src/views/Workspace/Cards/AddFindPlayground.jsx

```diff
@@ -0,0 +1,394 @@
+          .sort((a, b) => a.distance - b.distance)
+          .slice(0, 3);
+
+        this.setState({ results });
+    }
+
+    renderResults() {
+        const { classes } = this.props;
+        const { results } = this.state;
+
+        if (!results) return;
+
+        if (results.length === 0) return <div>Er zijn geen resultaten.</div>;
+
+        return (
+          <div>
+              <div className={classes.resultsTitle}>Zoekresulaten:</div>
+
+              {
+                  results.map(({ name, distance = 0, volunteerCount, id }) => {
+                      const convertedDistance = distance >= 1000
+                        ? `< ${Math.round(distance / 1000)} km`
+                        : `< ${Math.round(distance)} m`;
+
+                      return (
+                        <div key={name} className={classes.result}>
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Naam</div>
+                                <div className={classes.resultSubtitle}>{name}</div>
+                            </div>
+
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Afstand</div>
+                                <div className={classes.resultSubtitle}>{convertedDistance}</div>
+                            </div>
+
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Deelneemers</div>
+                                <div className={classes.resultSubtitle}>{volunteerCount}</div>
+                            </div>
+
+                            <Button
+                              className={classes.resultButton}
+                              color="primary"
+                              component={Link}
+                              to={`/workspace/${id}`}
+                              variant="contained"
+                            >Sluit je aan</Button>
+                        </div>
+                      );
+                  })
+              }
+
+              <Button
+                className={classes.addPlaygroundButton}
+                onClick={this.toggleAddPlayground}
+              >
+                  Mijn speeltuin staat er nog niet bij
+                  <ArrowIcon />
+              </Button>
+          </div>
+        );
+    }
+
+    render() {
+        const { classes, googleMapsKey } = this.props;
+        const { isAddPlaygroundOpen, userAddress } = this.state;
+
+        return (
+          <div>
+              <WorkspaceCard
+                title={"Speeltuin toevoegen"}
+                done={null}
+                image={require("../../../assets/img/backgrounds/workspace-welcome.jpg")}
+                content={"De eerste stap is om je aan te sluiten bij de actiepagina van jouw speeltuin, of om deze aan te maken. Vul je gegevens in om te beginnen."}
```

Comment:
De eerste stap is om te kijken of je je kunt aansluiten bij een bestaande actiepagina van jouw speeltuin. Misschien is een team bezig om jou speeltuin rookvrij te maken. Vul je plaatsnaam, adres of postcode in om te beginnen.

Reply:
Done